New README (#85)
meilisearch/meilisearch-go: README.md

  <a href="https://slack.meilisearch.com"><img src="https://img.shields.io/badge/slack-MeiliSearch-blue.svg?logo=slack" alt="Slack"></a>
</p>

> <p align="center">⚡ The MeiliSearch API client written for Golang</p>

Author: I also changed this sentence that I found unreadable. WDYT?

Reviewer: I agree, and just below we explain what MeiliSearch is, just in case!

Reviewer: I agree, and just below we explain what MeiliSearch is, just in case!

Author: @bidoubiwa what do you think about my change here? Is it weird to you like the description below?

> **MeiliSearch Go** is a client for **MeiliSearch** written in Go. **MeiliSearch** is a powerful, fast, open-source, easy to use and deploy search engine. Both searching and indexing are highly customizable. Features such as typo-tolerance, filters, facets and synonyms are provided out-of-the-box.

Reviewer 2: I read that MeiliSearch was written in Go 🤔. Not sure if it is me or if the english is weird.

Author: ```suggestion
**MeiliSearch Go** is the MeiliSearch API client for Go developers. **MeiliSearch** is a powerful, fast, open-source, easy to use and deploy search engine. Both searching and indexing are highly customizable. Features such as typo-tolerance, filters, facets and synonyms are provided out-of-the-box.
```

Author: WDYT @eskombro and @bidoubiwa?

Reviewer: I like it! 🚀

> **MeiliSearch Go** is a client for **MeiliSearch** written in Go. **MeiliSearch** is a powerful, fast, open-source, easy to use and deploy search engine. Both searching and indexing are highly customizable. Features such as typo-tolerance, filters, facets and synonyms are provided out-of-the-box.

Author: ```suggestion
**MeiliSearch Go** is the MeiliSearch API client for Go developers. **MeiliSearch** is a powerful, fast, open-source, easy to use and deploy search engine. Both searching and indexing are highly customizable. Features such as typo-tolerance, filters, facets and synonyms are provided out-of-the-box.
```

}
```

> With the `updateId`, you can check the status (`enqueued`, `processed` or `failed`) of your documents addition thanks to this [method](https://docs.meilisearch.com/references/updates.html#get-an-update-status).

Reviewer 2: ```suggestion
With the `updateId`, you can check the status (`enqueued`, `processed` or `failed`) of your documents addition using the [update endpoint](https://docs.meilisearch.com/references/updates.html#get-an-update-status).
```

}
```

## 🤖 Compatibility with MeiliSearch

This package only guarantees the compatibility with the [version v0.15.0 of MeiliSearch](https://github.com/meilisearch/MeiliSearch/releases/tag/v0.15.0).

## 💡 Learn More

The following sections may interest you:

> - **Manipulate documents**: see the [API references](https://docs.meilisearch.com/references/documents.html) or read more about the [document concept](https://docs.meilisearch.com/guides/main_concepts/documents.html).
> - **Search**: see the [API references](https://docs.meilisearch.com/references/search.html) or follow our [guide on search parameters](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html).
> - **Manage the indexes**: see the [API references](https://docs.meilisearch.com/references/indexes.html) or read more about the [index concept](https://docs.meilisearch.com/guides/main_concepts/indexes.html).
> - **Configure the index settings**: see the [API references](https://docs.meilisearch.com/references/settings.html) or follow our [guide on settings parameters](https://docs.meilisearch.com/guides/advanced_guides/settings.html#synonyms).

Reviewer: ```suggestion
- **Manipulate documents**: see the [API references](https://docs.meilisearch.com/references/documents.html) or read more about [documents](https://docs.meilisearch.com/guides/main_concepts/documents.html).
- **Search**: see the [API references](https://docs.meilisearch.com/references/search.html) or follow our [guide on search parameters](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html).
- **Manage the indexes**: see the [API references](https://docs.meilisearch.com/references/indexes.html) or read more about [indexes](https://docs.meilisearch.com/guides/main_concepts/indexes.html).
- **Configure the index settings**: see the [API references](https://docs.meilisearch.com/references/settings.html) or follow our [guide on settings parameters](https://docs.meilisearch.com/guides/advanced_guides/settings.html#synonyms).
```

Author: Waiting for @bidoubiwa's opinion! 🙂

Reviewer: Just as a note, notice that I removed the `#synonyms` from the last link, I think it should be removed in any case

Author: Oh yes! Well spotted!!

Reviewer 2: Both are nice but I prefer eskombro's for readability.
